src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandArtifact.java

```diff
@@ -21,7 +21,7 @@
  *
  * Created by tsato on 4/19/17.
  */
-public class StrandArtifact extends GenotypeAnnotation implements StandardSomaticAnnotation {
+public class StrandArtifact extends GenotypeAnnotation implements StandardSomaticAnnotation, StandardM2Annotation {
```

Comment:
You can delete `StandardSomaticAnnotation` entirely and absorb implementing classes into `StandardM2Annotation`.

Reply:
@davidbenjamin The only issue would seem to be that the UniqueAltReadCount appears to be a standard somatic annotation yet not included by default for M2, i'll add it to the standard M2 Annotations for now.

Reply:
I wouldn't do that @jamesemery -- we don't want to be changing the default Mutect annotations as part of this PR

Reply:
Regardless of that, `StandardSomaticAnnotation` doesn't seem to be very useful.  As far as I'm concerned you can delete it and leave `UniqueAltReadCount` without any extra annotation.